### DISCUSSION TITLE
Daily 16/05

### DISCUSSION THREAD

**Discussion by BraianTroncoso** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#295] [Daily Sprint]
- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Realicé las Issues que me habian asignado, me asigné otra mas que quedó libre en to do porque ya venia a full jaja, igual ahora tengo que refactorizar algunas cuestiones y volver a tirar un pull.
Hoy tengo clases de Ingles y tengo que realizar una clase de python que me quedó pendiente.
También estoy haciendo un curso de django principiantes.

**Comment by NataliaAlvarezIspc** [comment]
Ayer tuve que ocuparme de hacer una nueva tarea que no estaba prevista así que no pude continuar con el contenido de la plataforma.
Hoy voy a tratar de terminar con la tarea y con el contenido de la plataforma

**Comment by rpgrca** [comment]
Hoy tengo clases de Web y después voy a ver la oficina virtual. Ayer hice la edición de un elemento de envio y tipo de producto, quiero hacer lo de los productos también y demás partes del dashboard, y quiero ver si para mañana puedo armar un dashboard para el administrador que tenga todas las opciones de edición que hoy son partes separadas. También quiero ver si puedo hacer una de las tareas del ABM de django después de la explicación para terminar esa parte también.
